.ci/jobs/search-ui.yml

```diff
@@ -13,10 +13,9 @@
           repo: search-ui
           repo-owner: elastic
           branch-discovery: 'no-pr'
-          discover-pr-origin: current
+          discover-pr-origin: 'both'
           discover-pr-forks-strategy: 'merge-current'
           discover-pr-forks-trust: 'permission'
-          discover-pr-origin: 'merge-current'
```

Comment:
Duplicate keys were causing prettier to error out. 

I updated the config with the option that I thought made the most sense:
https://jenkins-job-builder.readthedocs.io/en/latest/project_githuborg.html

> discover-pr-origin (str) – Discovers pull requests where the origin repository is the same as the target repository. Valid options: merge-current, current, both, false. (default ‘merge-current’)